packages/core/ui-settings/core-ui-settings-server-internal/src/clients/base_ui_settings_client.ts

```diff
@@ -99,6 +100,20 @@ export abstract class BaseUiSettingsClient implements IUiSettingsClient {
     }
   }
 
+  private async getDefaultValues(context?: GetUiSettingsContext) {
+    const values: { [key: string]: unknown } = {};
+
+    for (const [key, definition] of Object.entries(this.defaults)) {
+      if (definition.getValue) {
+        values[key] = await definition.getValue(context);
```

Comment:
Worth a try/catch here and then fallback to a definition.value? Keep `definition.value` value required?

Reply:
`definition.value` is not required currently so not sure I would require it now. But we can fall back to that value if it is provided, or `undefined` if not. 👍

Reply:
Done in https://github.com/elastic/kibana/pull/187763/commits/56a895180a632d19a710ed2829057ad04ff67a5b